components/editor/modules/teaser/ui.js

```diff
@@ -46,7 +46,8 @@ const titleSizes = [
 
 const kinds = [
   { value: 'editorial', text: 'Editorial' },
-  { value: 'meta', text: 'Meta' }
+  { value: 'meta', text: 'Meta' },
+  { value: 'scribble', text: 'Ameise' }
```

Comment:
Really don't like the name as it sure as hell is going to confuse the first person that gets a Trello card "Ameisen fixen" and is not you or Thomas (or me, if I happen to remember, which I won't). Why not just name it "ant"? 
Neither of the two names is descriptive enough to be self-explanatory, so I'd just stick to what people call it anyway. Just my two cents.

Reply:
A verbal abstraction, so to speak.

Reply:
Yeah, I get your point. At the same time, on the code level I see value in having meaningful names that represent what something actually is (scribble: kurzes Geschreibsel, Gekritzel, Notiz). But I admit, that might be also driven by my internal resistance against naming stuff after insects.